src/main/hook/patch/gfx.h

```diff
@@ -26,4 +24,4 @@ void patch_gfx_init();
  */
 void patch_gfx_scale(enum patch_gfx_scale_mode scale_mode);
 
-void patch_gfx_frame_limit(uint16_t frame_limit);
\ No newline at end of file
+void patch_gfx_display(uint8_t scaling_mode, uint8_t resizable_window, uint16_t screen_width, uint16_t screen_height, uint16_t frame_limit);
```

Comment:
```suggestion
void patch_gfx_display(enum patch_gfx_scale_mode scaling_mode, bool resizable_window, uint16_t screen_width, uint16_t screen_height, uint16_t frame_limit);
```

Some type improvements.

Also, why did you merge frame limit and scaling? Can these two options not live separately on the module interface? Creates a cleaner interface imo.